gammapy/spectrum/tests/test_spectrum_extraction.py

```diff
@@ -84,6 +84,19 @@ def test_spectrum_extraction(pars,results,tmpdir):
     assert_allclose(ana.observations[0].lo_threshold,results['ethresh'])
 
     assert_quantity_allclose(obs23523.aeff.evaluate(energy=5*u.TeV),results['aeff'])
+    
+    # Write on set of output files to gammapy-extra as input for other tests
+    # and check I/O
+    if not pars['containment_correction']:
```

Comment:
@JouvinLea 

I added this in order to automatically write the example OGIP files in gammapy-extra, so we don't have to copy them there everytime we change something.

What do you think? ok?